templates/includes/_info_messages.html

```diff
@@ -0,0 +1,8 @@
+ {% for message in messages %}
+    <div class="alert alert-info alert-dismisable" role="alert" style="margin-top:10px;margin-bottom:10px;">
```

Comment:
Do you need the style specifications here? I took it out and it looked the same...